elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -329,8 +356,8 @@
       }
     }
     
-    &:hover,
-    &:focus {
+    &:focus,
```

Comment:
@kelsS so I'm getting that this is an intentional order, you know something I don't?

Reply:
@wesruv Oh yeah, read something about the focus/hover CSS rule order on MDN a while back and just started doing it the way the docs said it should be done and now it is a habit. I think it is discussed in this [MDN article](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Styling_links), not sure how much issue it will actually cause though in a different order in practice. The article is vague about the issues it will cause, it just says "If you put these in the wrong order, things won't work properly." lol Let me know your thoughts on that article.

Reply:
Muchos dankes